docs/new_instrument.rst

```diff
@@ -26,18 +33,26 @@ Pysat maintains a list of files to enable data management functionality. It need
 
 where tag indicates a specific subset of the available data from cnofs_vefi.
 
-See pysat.utils.create_datetime_index for creating a datetime index for an array of irregularly sampled times.
+See pysat.utils.time.create_datetime_index for creating a datetime index for an array of irregularly sampled times.
 
-Pysat will store data in pysat_data_dir/platform/name/tag, helpfully provided in data_path, where pysat_data_dir is specified by user in pysat settings.
+Pysat will store data in pysat_data_dir/platform/name/tag, helpfully provided in data_path, where pysat_data_dir is specified by using `pysat.utils.set_data_dir(pysat_data_dir)`.
 
-pysat.Files.from_os is a convenience constructor provided for filenames that include time information in the filename and utilize a constant field width. The location and format of the time information is specified using standard python formatting and keywords year, month, day, hour, minute, second. A complete list_files routine could be as simple as
+`pysat.Files.from_os` is a convenience constructor provided for filenames that include time information in the filename and utilize a constant field width. The location and format of the time information is specified using standard python formatting and keywords year, month, day, hour, minute, second. A complete list_files routine could be as simple as
 
 .. code:: python
 
    def list_files(tag=None, data_path=None):
-       return pysat.Files.from_os(data_path=data_path, 
+       return pysat.Files.from_os(data_path=data_path,
                     format_str='cindi-{year:4d}{day:03d}-ivm.hdf')
-                                
+
+The list_files function can be invoked from within ipython by calling
```

Comment:
Technically the information in instrument.files.files comes from list_files, but you aren't directly invoking the method.

If a user wants pysat to run list_files when the Instrument is instantiated, then pass update_files=True.

Reply:
added some text about this